src/main/java/rotateit/service/github/label/OnLabelEventToPersistListener.java

```diff
@@ -0,0 +1,14 @@
+package rotateit.service.github.label;
+
+import org.springframework.context.event.EventListener;
+import org.springframework.stereotype.Component;
+import rotateit.domain.Event;
+import rotateit.domain.github.label.LabelPayload;
+
+@Component
```

Comment:
Actually this bean wasn't declared in context. And scanning is enabled by `@SpringBootApplication` annotation.